### PR TITLE
Alerts | Stop-Level

### PR DESCRIPTION
**Pull request by i0sea**
#### Summary of changes

**Asana Ticket:** [Alerts | Stop-Level](https://app.asana.com/0/385363666817452/1204305469426563/f)

Add alerts banner of alerts related to stop
Screen Shot 2023-04-27 at 10 24 08 AM

---

#### General checks
* [ ] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [ ] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [ ] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

### CODE REVIEW

**Comment by arkadyan** [comment]
2 immediate notes before even looking at the code:

1. Could you please squash the commits? At a glance, it looks like a single commit would be appropriate for this change.
2. You probably already noticed this, but there's a failing TS check.

**Comment by i0sea** [comment]
Squashed. The lint issue is unrelated to the changes here but has `/* istanbul ignore next */` might need an "ts-ignore` as well

**Comment by arkadyan** [comment]
@i0sea Thanks! Sounds like that "ignore" fix could be an easy PR that would get things passing again.

File: apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx [hunk 1]
```diff
@@ -29,4 +30,29 @@ describe("StopPageRedesign", () => {
...
+
+    jest
+      .spyOn(useAlertsForStop, "default")
+      .mockImplementation(() => [lowAlert]);
+
+    render(<StopPageRedesign stopId="123" />);
+    expect(
+      screen.queryByText("There is construction at this station.")
+    ).not.toBeNull();
+  });
```
Comment (arkadyan) [inline apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx:57]: suggestion (non-blocking): If you have an interest in additional test for behavior you are already handling, you could add one where `useAlertsForStop` returns `null` and the page still displays properly.

**Review by arkadyan — APPROVED** [review]
👍 Nice, seems pretty straightforward.

File: apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx (hunk 1 above)
Comment (kotva006) [reply apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx:102]: The endpoint should be returning an empty list if there are no alerts.  Undefined or null should be an error state, or we are waiting for data to load

**Pull request merged**